locales/zh-CN/security.ftl

```diff
@@ -4,19 +4,77 @@
+                    <li><a href="http://rustup.rs">rustup.rs</a></li>
+                    <li><a href="http://crates.io">crates.io</a>（见下文例外情况）</li>
+                    <li><a href="http://docs.rs">docs.rs</a></li>
+                    <li><a href="http://rfcbot.rs">rfcbot.rs</a></li>
+                </ul>
+            </li>
+            <li>在 <a href="https://crates.io">crates.io</a> 上由 <a href="https://crates.io/users/rust-lang-owner">@rust-lang-owner</a> 拥有的所有 crate。</li>
+            <li>在 Visual Studio Marketplace 上由 <a href="https://marketplace.visualstudio.com/publishers/rust-lang"><code>rust-lang</code></a> 发布的所有扩展。</li>
+            <li>在 Open VSX 注册表中由 <a href="https://open-vsx.org/namespace/rust-lang"><code>rust-lang</code></a> 发布的所有扩展。</li>
+        </ul>
+        <p>以下内容<strong>不在我们的范围内</strong>：</p>
+        <ul>
+            <li><a href="http://internals.rust-lang.org">internals.rust-lang.org</a> 和 <a href="http://users.rust-lang.org">users.rust-lang.org</a> 域名。请遵循 <a href="https://github.com/discourse/discourse/blob/main/docs/SECURITY.md">Discourse 的安全政策</a>。</li>
+            <li>在 <a href="http://crates.io">crates.io</a> 上发布的第三方包。请遵循 <a href="https://crates.io/security">crates.io 的安全政策</a>。</li>
+        </ul>
+        <p>报告漏洞时，请记住：</p>
+        <ul>
+            <li>除非另有说明，Rust 工具链的所有组件（rustc、Cargo、rust-analyzer 或通过 rustup 提供的任何其他工具）假定用户的源代码和依赖项是完全可信的、经过审查且不包含恶意代码。我们不认为因编译或分析恶意项目或依赖项而导致的攻击是安全漏洞。</li>
+            <li>Rust 编译器或语言中的健全性问题不会自动归类为安全漏洞，但如果报告，将根据具体情况进行分析。</li>
+            <li><code>regex</code> crate <a href="https://docs.rs/regex/latest/regex/#untrusted-input">对不受信任的模式提供保证</a>。我们仅在 <code>regex</code> crate 内的时间不是线性时，且 <a href="https://docs.rs/regex/latest/regex/struct.RegexBuilder.html"><code>RegexBuilder</code> 中的限制方法</a>均无法防止攻击时，才将使用不受信任模式导致的拒绝服务视为安全漏洞。</li>
+        </ul>
+        <p>如果您对某些内容是否属于我们的范围有疑问，请<a href="mailto:security@rust-lang.org">联系我们</a>，我们将提供指导。</p>
+
 security-disclosure-heading = 信息披露政策
-security-disclosure-description =
-    <p>Rust 项目有 5 步披露流程：</p>
-    <ol>
-      <li>收到安全报告后，将其分配给一位主要处理人。此人将协调修复和发布过程。</li>
-      <li>确认问题并确定所有受影响版本。</li>
-      <li>审核代码以发现任何潜在的类似问题。</li>
-      <li>对所有仍在维护的版本准备修复程序。这些修复程序将不会推送到公共仓库，而是保留在本地，等待发布。</li>
-      <li>过封锁期后，<a href="{ -rustlang-security-announcements-google-groups-forum-href }">Rust 安全邮件列表</a>将发布一份公告副本。这些更改会提交至公共仓库，更新后的程序将部署至 rust-lang.org。在收到通知邮件列表的 6 小时内，Rust 博客将发布该通报的副本。</li>
-    </ol>
+security-disclosure-description--2025-07 =
+        <p>Rust 项目有 5 步披露流程：</p>
+        <ol>
+          <li>收到安全报告后，将其分配给一位主要处理人。此人将协调修复和发布过程。</li>
+          <li>确认问题并确定所有受影响的版本，并邀请相关 Rust 团队的领域专家参与。</li>
+          <li>审核代码以发现任何潜在的类似问题。</li>
+          <li>为所有受支持的发布分支准备修复程序，并保留 CVE 编号。这些修复程序不会提交到公共仓库，而是保存在私有仓库中，等待公告发布。这些修复程序会使用与公共更改相同的审查流程进行私下审查。</li>
+          <li>在封禁日期，公告的副本会发送到 <a href="{ -rustlang-security-announcements-google-groups-forum-href }">Rust 安全邮件列表</a> 并发布在 Rust 博客上。这些更改会推送到公共仓库，并启动发布流程。在一小时内，CVE 数据库中会发布完整的详细信息。</li>
+        </ol>
```

Comment:
Nit:

```suggestion
          <li>过封锁期后，公告的副本会发送到 <a href="{ -rustlang-security-announcements-google-groups-forum-href }">Rust 安全邮件列表</a> 并发布在 Rust 博客上。这些更改会推送到公共仓库，并启动发布流程。在一小时内，CVE 数据库中会发布完整的详细信息。</li>
        </ol>
```


“在封禁日期”不准确，因为这个日期恰恰是**解禁**日期，即 $\text{原来封禁} \to \text{现在解禁}$。原来的“过封锁期后”是更准确的。或者“在解除封锁当天”也行。
"On the day of embargo" isn't quite accurate, because that date is the date when the embargo is lifted, i.e. previously embargoed -> now lifted. The original "after embargo duration" is more accurate. Alternatively, "on the day of lifting the embargo" works too.